ci_framework/roles/devscripts/tasks/cleanup.yml

```diff
@@ -29,6 +29,14 @@
     chdir: "{{ cifmw_devscripts_repo_dir }}"
     target: clean
 
+- name: Remove the SSH key generated for accessing the platform
+  ansible.builtin.file:
+    path: "{{ cifmw_devscripts_artifacts_dir }}/{{ item }}"
+    state: absent
+  loop:
+    - "{{ lookup('env', 'USER') }}_ed25519"
```

Comment:
lookup() is local - so this will match the user on the ansible controller running the playbook.

You want to use `ansible_user_id` here imho

Reply:
Thank you. I need to run these remotely. It should be fixed in the latest commit.